index.html

```diff
@@ -0,0 +1,1249 @@
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>Floating badges</h2>
+                    <div class="badge-container">
+                        <div class="badge">
+                            <button type="button" class="btn btn-default">
+                                Notifications<span class="badge-float ">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-primary">
+                                Notifications<span class="badge-float  badge-secondary">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-success">
+                                Notifications<span class="badge-float badge-info">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-secondary">
+                                Notifications<span class="badge-float badge-success">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-danger">
+                                Notifications<span class="badge-float badge-warning">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-info">
+                                Notifications<span class="badge-float badge-default">23</span>
+                            </button>
+                        </div>
+
+                        <div class="badge">
+                            <button type="button" class="btn btn-warning">
+                                Notifications<span class="badge-float badge-danger">23</span>
+                            </button>
+                        </div>
+
+
+                    </div>
+                    <iframe

```

Comment:
some of the carbon snippets have a vertical scroll, is it a feature? or a bug? Nevertheless, it doesn't look great

Reply:
the code of the snippets is large so that scroll down comes